sdk/nodejs/npm/npm.go

```diff
@@ -101,3 +103,14 @@ func (node *npmManager) Pack(ctx context.Context, dir string, stderr io.Writer)
 
 	return packTarball, nil
 }
+
+func (node *npmManager) LinkPackages(ctx context.Context, packages map[string]string) error {
+	for packageName, packagePath := range packages {
+		packageSpecifier := fmt.Sprintf("dependencies.%s=file:%s", packageName, packagePath)
+		cmd := exec.Command(node.executable, "pkg", "set", packageSpecifier) //nolint:gosec
```

Comment:
We're not setting working dir here. Pretty sure this will do the wrong thing in conformance tests and also in normal programs if "main" isn't the same as the root dir.